src/main/java/org/forgerock/openam/examples/SampleAuth.java

```diff
@@ -42,67 +42,72 @@
 import com.sun.identity.shared.datastruct.CollectionHelper;
 import com.sun.identity.shared.debug.Debug;
 
-
-
-public class SampleAuth extends AMLoginModule
-{
+/**
+ * SampleAuth authentication module example.
+ *
+ * If you want to create your own module based on this example, you will need to modify every "SampleAuth" occurrences
```

Comment:
We'll want to put a copy of the file in the _Dev Guide_, <http://openam.forgerock.org/doc/bootstrap/dev-guide/index.html#authentication-logic-sample-auth-module>.

You might want to try to fold lines at around the 80th column, because long lines overflow off the edge of the page in PDF.